accounts/views.py

```diff
@@ -47,6 +47,82 @@ def get(self, request):
         return HttpResponseServerError()
 
 
+class DevLoginView(View):
+    """
+    Log in a test user.
+    Does not use Shibboleth
+    """
+
+    @staticmethod
+    def populate_user_data():
+        NAMES = [
+            "George Washington",
+            "John Adams",
+            "Thomas Jefferson",
+            "James Madison",
+            "James Monroe",
+            "John Quincy Adams",
+            "Andrew Jackson",
+            "Martin Van Buren",
+            "William Harrison",
+            "John Tyler",
+        ]
+        users = []
+        for i, name in enumerate(NAMES):
+            first, last = name.split(" ", 1)
+            pennkey = first.lower() + last[0].lower()
+            affiliation = "student;member"
+            users.append(
+                {
+                    # if leave as i, g. wash's pennid is 0, which makes not remote_user true
+                    "pennid": i + 1,
+                    "pennkey": pennkey,
+                    "first_name": first,
+                    "last_name": last,
+                    "affiliation": affiliation,
+                }
+            )
+        return users
+
+    def get(self, request):
+        users = DevLoginView.populate_user_data()
+        return render(request, "accounts/devlogin.html", {"users": users})
```

Comment:
Once we create the users in the database, we can just use a queryset here (like Users.objects.all()) instead of a function call to get the users for the view